.github/release-drafter.yml

```diff
@@ -0,0 +1,51 @@
+name-template: "$RESOLVED_VERSION"
+tag-template: "$RESOLVED_VERSION"
+
+categories:
+  - title: "Backwards Incompatible Changes"
+    labels:
+      - "backwards incompatible"
+
+  - title: "Features"
+    labels:
+      - "feature"
+      - "enhancement"
+      # usually combined with enhancement (avoid double listing)
+      - "deprecation"
+
+  - title: "Bug Fixes"
+    labels:
+      - "bug"
+      - "bug fix"
+
+  - title: "Other Changes"
+    labels:
+      - "dependency"
+      - "docs"
+      - "trivial"
+      - "maintenance"
+
+change-template: "- $TITLE (#$NUMBER) @$AUTHOR"
+exclude-labels:
+  - "skip-changelog"
+template: |
+  $CHANGES
+
+version-resolver:
+  major:
+    labels:
+      - "backwards incompatible"
+  minor:
+    labels:
+      - "feature"
+      - "enhancement"
+      - "deprecation"
+  patch:
+    labels:
+      - "bug fix"
+      - "bug"
+      - "dependency"
+      - "docs"
+      - "maintenance"
```

Comment:
The "maintenance" tag is not user-facing, so it could be excluded from "Other changes".